### ISSUE TITLE
Second window to open in NSLogger.app is missing content

### ISSUE THREAD

**Issue by cssbz** [fpillet/NSLogger#199]
I have several .nsloggerdata files saved.

If I open more than one at once, the second window to open lacks some content.  In the screenshot below, there are several problems with the window on the right, which I opened second.

![screen shot 2016-05-02 at 21 04 07](https://cloud.githubusercontent.com/assets/945402/14966467/6f9ee1f4-10aa-11e6-8b22-d489f017476b.png)

The window on the right:
- has no vertical scroll bar
- has no bar along the bottom (which should show the number of messages etc)
- isn't wrapping the text (note that the two windows are showing the same log content... I've lined up the two "Updated expiryDate" entries).

Here are some things I've noticed about when this occurs (I'm relaunching NSLogger.app before each step 1):

```
1. Open A.nsloggerdata.  Window is OK.
2. Open B.nsloggerdata.  Window has issues.

1. Open B.nsloggerdata.  Window is OK.
2. Open A.nsloggerdata.  Window has issues.

1. Open A.nsloggerdata.  Window is OK.
2. Open B.nsloggerdata.  Window has issues.
3. Open C.nsloggerdata.  Window has issues.

1. Open A.nsloggerdata.  Window is OK.
2. Close the A.nsloggerdata window.
3. Open B.nsloggerdata.  Window is OK.
```

I've been experiencing this for quite a while, but have only just figured out how to reliably reproduce it.  It has quite an impact on usability.  It might be the same as https://github.com/fpillet/NSLogger/issues/194, but I'm not sure (I haven't seen NSLogger.app crash).

I'm using NSLogger.app 1.6.0.

**Comment by fpillet** [comment]
Thanks for the detail repro instructions, Ashley! Yes this is the same issue as #194 which is started to get on my nerves to, so I hopefully will get this fixed very soon.

**Comment by cssbz** [comment]
Great, thanks! Sorry I'm not trying to wrangle a fix myself this time.

**Comment by 0xced** [comment]
I quickly looked at that bug and it seems that the `mainDisplay` frame is not computed correctly at [LoggerWindowController.m:631](https://github.com/fpillet/NSLogger/blob/6eeaf6d4e67107600281c29f171eb2ff4b1eb6f8/Desktop%20Viewer/Classes/LoggerWindowController.m#L631)
I changed this:
```objc
frame.size.height += newSize.height - oldSize.height;
```
to this:
```objc
frame.size.height = newSize.height;
```
then the bottom bar is displayed properly for the second window too.

This change also gets rid of the following warning that I was seeing:
```
2016-05-23 23:36:42.896 NSLogger[20613:1901927] <LoggerSplitView: 0x10061f390>: the delegate <LoggerSplitView: 0x10061f390> was sent -splitView:resizeSubviewsWithOldSize: and left the arranged view frames in an inconsistent state:
2016-05-23 23:36:42.896 NSLogger[20613:1901927] Split view bounds: {{0, 0}, {885, 673}}
2016-05-23 23:36:42.896 NSLogger[20613:1901927]     Arranged view frame: {{0, 0}, {397, 673}}
2016-05-23 23:36:42.896 NSLogger[20613:1901927]     Arranged view frame: {{398, 0}, {987, 507}}
2016-05-23 23:36:42.896 NSLogger[20613:1901927] The outer edges of the arranged view frames are supposed to line up with the split view's bounds' edges. NSSplitView is working around the problem, perhaps at the cost of more redrawing. (This message is only logged once per NSSplitView.)
```

**Comment by fpillet** [comment]
Thanks, Cédric - adding the fix to current codebase!

**Comment by 0xced** [comment]
There’s still a glitch where the green header (first message) is not properly centered. This can be worked around by manually resizing the horizontal split view.

**Comment by fpillet** [comment]
Yes I'm looking into it. It looks like the tableView is set to the full window width, not taking the sidebar into account. This also affects log text wrapping.

**Comment by fpillet** [comment]
Fixed for 1.8.0
